### PR TITLE
fix: Apply Code Ui Changes

### PR DESCRIPTION
**Pull request by iamdevvalecha**
#7265 
Fixes:
Ui part of promo code to buy tickets.

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
The Changes:-
Screenshot from 2021-05-12 22-21-40

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -165,11 +165,11 @@
                 @key-up={{action "handleKeyPress"}}
                 @id="promotionalCode" />
               {{#unless this.promotionalCodeApplied}}
-                <div role="button" class="ui icon {{if this.promotionalCode '' 'disabled'}} button" {{action 'applyPromotionalCode'}}>
-                  <i class="checkmark icon"></i>
+                <div role="button" style="height: 46px" class="ui icon {{if this.promotionalCode '' 'disabled'}} button" {{action 'applyPromotionalCode'}}>
+                {{t 'Apply Code'}}
```
Comment (MeghalBisht) [inline app/templates/components/public/ticket-list.hbs:169]:
```suggestion
                {{t 'Apply code'}}
```

File: app/templates/components/public/ticket-list.hbs [hunk 2]
```diff
@@ -165,11 +165,11 @@
                 @key-up={{action "handleKeyPress"}}
                 @id="promotionalCode" />
               {{#unless this.promotionalCodeApplied}}
-                <div role="button" class="ui icon {{if this.promotionalCode '' 'disabled'}} button" {{action 'applyPromotionalCode'}}>
-                  <i class="checkmark icon"></i>
+                <div role="button" style="height: 46px" class="ui icon {{if this.promotionalCode '' 'disabled'}} button" {{action 'applyPromotionalCode'}}>
```
Comment (MeghalBisht) [inline app/templates/components/public/ticket-list.hbs:168]: This is not a good approach. What if I change the height of the parent div someday? 
Try height 100%

File: app/templates/components/public/ticket-list.hbs [hunk 3]
```diff
@@ -165,11 +165,11 @@
...
+                {{t 'Apply Code'}}
                 </div>
               {{/unless}}
-              <div role="button" class="ui black icon button" {{action 'togglePromotionalCode'}}>
+              <div role="button" class="ui black icon button" style="height: 46px" {{action 'togglePromotionalCode'}}>
```
Comment (MeghalBisht) [inline app/templates/components/public/ticket-list.hbs:172]: This is not a good approach. What if I change the height of the parent div someday? 
Try height 100%

**Review by MeghalBisht — APPROVED** [review]

**Comment by iamdevvalecha** [comment]
thanks :)

**Pull request merged**